src/Auth.ts

```diff
@@ -68,9 +75,8 @@ export class Auth {
   }
 
   constructor() {
-    this.appId = config.cliAadAppId;
     this._service = new Service();
-    this.authCtx = new AuthenticationContext(`https://login.microsoftonline.com/${config.tenant}`);
+    this.authCtx = new AuthenticationContext(`https://login.microsoftonline.com/${this.service.tenant}`);
```

Comment:
Would it make sense to call `setAuthContext()` here too?

Reply:
Good catch!